lib/trento_web/openapi/v1/schema/cluster.ex

```diff
@@ -135,7 +130,7 @@ defmodule TrentoWeb.OpenApi.V1.Schema.Cluster do
         type: %Schema{
           type: :string,
           description: "Detected type of the cluster",
-          enum: ClusterType.values()
+          enum: [:hana_scale_up, :hana_scale_out, :unknown]
```

Comment:
Using hardcoded values as they cannot be changed without adding non backward compatible changes.
This schema won't be updated anymore